app/policies/students_group_policy.rb

```diff
@@ -0,0 +1,20 @@
+class StudentsGroupPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+
+  def index?
+    true
+  end
+
+  def create?
+    true
+  end
+
+
+  def destroy?
```

Comment:
in this case, record is the student_group
student_group is not connected to user directly, but only via students
so it should be record.student.user = user

Reply:
ok I trust you on that, Carlos was doing this with me and he didn't notice that.. but he was a bit distracted so let's do it your way, I am a bit bad with pundit and devise, still learning. Can we test that somehow later?

Reply:
sure! I get you, I've only discovered this because of my own errors. Once we've implement the destroy action to the students_group we can write record.user = user again, try to destroy and see what happens